source/patterns/@aws-solutions-constructs/core/index.ts

```diff
@@ -13,39 +13,44 @@
 
 export * from './lib/apigateway-defaults';
 export * from './lib/apigateway-helper';
+export * from './lib/cloudfront-distribution-defaults';
```

Comment:
Seems like lot of unnecessary arrangements

Reply:
Updating to add my changes to the bottom.